packages/sdk/src/extensions/integrations/instances/erc4626.test.ts

```diff
@@ -0,0 +1,158 @@
+import { ALICE, BOB, ERC4626_ADAPTER, INTEGRATION_MANAGER, MA_WETH, WETH } from "../../../../tests/constants.js";
+import { publicClient, sendTestTransaction, testActions } from "../../../../tests/globals.js";
+import { toWei } from "../../../utils/conversion.js";
+import { multiplyBySlippage } from "../../../utils/slippage.js";
+import { Integration } from "../integrationTypes.js";
+import { prepareUseIntegration } from "../prepareUseIntegration.js";
+import { parseAbi } from "viem";
+import { test } from "vitest";
+
+const abiMaConversion = parseAbi([
```

Comment:
minor thing but I tend to name rather what contract/standard they represent. So here I would name it `abiERC4626`, as here we represent the standard https://eips.ethereum.org/EIPS/eip-4626